app/md/sponsors/prospectus.md

```diff
@@ -53,52 +52,100 @@ As a sponsor, your funding assistance will serve three critical purposes:
-1. Build a better PyTexas by enabling organizers to attract attendees with standard conference perks such as printed t-shirts, swag bags, catering, professional A/V, and more.
-1. Help keep the cost of PyTexas low. PyTexas 2017 attendees will pay a modest 
-    registration fee, and almost all of this fee will go to the rental of our 
-    venue. Your sponsorship helps us keep this price reasonable for everyone 
-    involved.
+1. Build a better PyTexas by enabling organizers to attract attendees with
+    standard conference perks such as printed t-shirts, catering,
+    professional A/V, and more.
+1. Help keep the cost of PyTexas low. PyTexas 2017 attendees will pay a modest
+    registration fee of $85. Your sponsorship helps us keep this price
+    reasonable for everyone involved.
 
 ## Sponsorship Opportunities
 
-We offer several standard packages. However, if none of these seem to fit, 
+We offer several standard packages. However, if none of these seem to fit,
 please feel free to contact the Sponsor Chair (above). We want to help you make the
 most of your investment in the Python community.
 
-*More Info to Come Soon*
+### Standard Packages
+
+- Bronze $300
+    - Available for individuals and startups of up to 10 employees
+    - Name and link included on PyTexas website
+    - Company name and link thanked in PyTexas twitter feed
+    - 1 free ticket to PyTexas
+- Silver $750
+    - Name, link, and logo included on the PyTexas website
+    - Promotional items placed in the PyTexas swag table
+    - Company name and link included in the PyTexas twitter feed and social platforms
+    - 1 free tickets to PyTexas
+- Gold $1200
+    - All the benefits of Silver-level sponsorship
+    - Logo placement on slide decks
+    - 1 additional free ticket to PyTexas (2 total)
+- Platinum $1,500
+    - Limit: 5
+    - All the benefits of Gold-level sponsorship
+    - Table/Booth Space in Sponsor area
+    - Company banner placed in the large conference room
+    - 1 additional free ticket to PyTexas (3 total)
+- Diamond $2,500
+    - Limit: 2
+    - All the benefits of Platinum-level sponsorship
+    - Premier brand placement for entire conference
+    - Branding of one room
+    - Company logo on PyTexas t-shirt (Size & placement subject to artistic
+      license of PyTexas organizers.)
+
+### Exclusive Benefits
+
+PyTexas also offers several exclusive sponsor benefits on a first-come, first-served basis:
+
+- Lanyard
+    - One slot available
+    - Sponsor will provide printed branded lanyards to PyTexas
+    (Need at least two weeks prior to event)
+- Lanyard targeted
+    - One slot available\=
+    - Each attendee’s badge will be affixed with a ribbon with a design of your
+    choice (subject to approval by Sponsor Chair)
+- Exclusive party slot ($1000)
+    - Saturday and Sunday slots currently available
+    - Promote your party on the day of the party
+    - Official promotion via website, schedule, and announcements
```

Comment:
I was thinking we could also add "Coffee Sponsor" in here as well. One slot for both days, they provide the coffee?